hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -45,6 +45,7 @@
 BANKS_TABLE = os.environ["BANKS_TABLE"]
 COUNTS_TABLE_NAME = os.environ["COUNTS_TABLE_NAME"]
 BANKS_MEDIA_BUCKET_NAME = os.environ["BANKS_MEDIA_BUCKET_NAME"]
+LCC_DURABLE_FS_PATH = os.environ["LCC_DURABLE_FS_PATH"]
```

Comment:
This will throw a `KeyError` unless you provide a value for LCC_DURABLE_FS_PATH here. https://github.com/facebook/ThreatExchange/blob/main/hasher-matcher-actioner/terraform/api/main.tf#L36-L51

Talk to @Samyakk123. He can help you get the value of `durable_storage_path` into your the terraform config at  api/main.tf.

See here: https://github.com/facebook/ThreatExchange/blob/main/hasher-matcher-actioner/terraform/main.tf#L425